aws_signing_helper/signer.go

```diff
@@ -129,6 +106,28 @@ var ignoredHeaderKeys = map[string]bool{
 	"X-Amzn-Trace-Id": true,
 }
 
+// Find whether the current certificate matches the CertIdentifier
+func certMatches(certIdentifier CertIdentifier, cert *x509.Certificate) bool {
+	// certMatches := true
```

Comment:
I think it may be necessary to rebase with the branch used for certificate store integration to remove these commented lines. I had commented this out for some local testing.

Reply:
Did a rebase before the push. These lines are not commented anymore.